language/tools/move-cli/src/test.rs

```diff
@@ -135,3 +135,24 @@ pub fn run_one(args_path: &Path, cli_binary: &str) -> anyhow::Result<()> {
         Ok(())
     }
 }
+
+pub fn run_all(args_path: &Path, cli_binary: &str) -> anyhow::Result<()> {
+    let mut test_total = 0;
+    let mut test_passed = 0;
+    for entry in datatest_stable::utils::iterate_directory(args_path) {
```

Comment:
I'd use `move_lang::find_filenames`. We already indirectly use that logic in the CLI and I think it will make things easier for this

Reply:
Thanks. I'll make the change.

Reply:
Second thought: `find_filenames` is not `pub` in `lib.rs`, shall I make it `pub`?

Reply:
> shall I make it pub?

Go for it